components/BrandingContent/FontSection/fonts.css

```
@@ -0,0 +1,21 @@
+.brandingFonts {
+  text-align: center;
+  margin: 3em 0;
+}
+
+.brandingFonts a {
+  color: #249cbc;
```

Can you make this change on any place where a hex color is hard-coded?
```suggestion
  color: var(--primary);
```